ui/components/resolvers/ResolverDetail.js

```diff
@@ -182,9 +192,20 @@ class ResolverDetail extends Component {
                     >
                         Save
                     </Button>
-                    {/* <Button bsStyle="default" onClick={() => console.log("cancel")}>Cancel</Button> */}
+                    <Button
+                        bsStyle="default"
+                        onClick={() => this.removeResolver()}
+                    >
+                        Delete Resolver
+                    </Button>
                 </div>
 
+                <DeleteResolverDialog
+                    showModal={showDeleteModal}
+                    resolver={resolver}
+                    onClose={() => this.setState({ showDeleteModal: false })}
+                    onDelete={() => this.onDeleteResolver()}
```

Comment:
Do we really need two different callbacks?

Reply:
The onclose one only hides the modal. The ondelete happens after the delete resets the entire state which basically unselects the resolver and shows the empty screen on the right. I think this is actually ok no what do you think? if it is I will just use the one function and reset the state If they press cancel (which also closes the modal)

Reply:
It's ok, two callbacks make sense here.

Reply:
👍